scripts/install_third_party.py

```diff
@@ -330,11 +332,38 @@ def download_manifest_files(filepath):
                     dependency_tar_root_name, dependency_target_root_name)
 
 
+def remove_emojis_from_guppy(guppy_file_path):
+    """The emoji symbols present in the guppy.min.js file are not compatible
+    with all platforms, hence they must be removed from the file. Leaving these
+    leads to console warnings which cause the e2e tests to fail.
+    For reference: https://github.com/daniel3735928559/guppy/issues/138
+
+    Args:
+        guppy_file_path: str. Path of the guppy.min.js file.
+    """
+
+    text_to_remove = (
+        u'utf8chars:{template:"utf8char",values:{banana:{char:"🍌"},pineapple:'
+        u'{char:"🍍"},mango:{char:"🥭"},kiwi:{char:"🥝"}}},')
+
+    with codecs.open(guppy_file_path, mode='r', encoding='utf-8') as f:
+        file_content = f.read()
+
+    file_content = file_content.replace(text_to_remove, u'')
+
+    with codecs.open(guppy_file_path, mode='w', encoding='utf-8') as f:
+        f.write(file_content)
+
+
 def main(args=None):
     """Installs all the third party libraries."""
     unused_parsed_args = _PARSER.parse_args(args=args)
     download_manifest_files(MANIFEST_FILE_PATH)
 
+    guppy_file_path = os.path.join(
+        THIRD_PARTY_STATIC_DIR, 'guppy-f944022', 'build', 'guppy.min.js')
```

Comment:
Move the file path generation code inside `remove_emojis_from_guppy` and add a comment here explaining the reason for this removal!

Reply:
While testing, the path is different which is why it's been passed as a parameter instead of hard-coding it inside the function.
Also, the docstring of the `remove_emojis_from_guppy` function explains the reason for this removal along with the relevant link to the issue filed in guppy's issue tracker.